src/PKSim.Infrastructure/ProjectConverter/v9_0/Converter80To90.cs

```diff
@@ -135,5 +137,16 @@ private void convertIndividualValueCache(IndividualValuesCache individualValuesC
          individualValuesCache.IndividualIds.AddRange(Enumerable.Range(0, firstParameterValue.Count));
          _converted = true;
       }
+
+      public void Visit(PopulationAnalysisChart populationAnalysisChart)
```

Comment:
All right. We have renamed the race into population. Analysis referencing race however are not happy. So here we go:)